apps/studio/components/interfaces/Support/AIAssistantOption.tsx

```diff
@@ -42,16 +39,8 @@ export const AIAssistantOption = ({
     return null
   }
 
-  const getProjectRef = () => {
-    if (projectRef !== 'no-project') {
-      return projectRef
-    }
-    // If no specific project selected, use first project from the org
-    const orgProjects = projects?.filter((p) => p.organization_slug === organizationSlug)
-    return orgProjects?.[0]?.ref || '_'
-  }
-
-  const aiLink = `/project/${getProjectRef()}?aiAssistantPanelOpen=true`
+  // If no specific project selected, use the wildcard route
+  const aiLink = `/project/${projectRef !== 'no-project' ? projectRef : '_'}?aiAssistantPanelOpen=true`
```

Comment:
This is a change in behaviour, it used to preselect the first project in the org but now it always sends you to select a project.

Reply:
yup! i was thinking this might make more sense actually since we're only sending users to the wildcard route if "no specific project" is selected

e.g if i selected "no specific project" and I click on the CTA to use the AI assistant, then i get sent to (from the user's POV at least despite it being the first project in the org) a random project to use the assistant

although what do you reckon though? happy to revert this nonetheless if you reckon the original behaviour is better 🙏